### ISSUE TITLE
Keras: how to define custom loss function for a ragged tensor output with shape(None, 1)

### ISSUE THREAD

**Issue by zwh14** [tensorflow/tensorflow#53598]
Suppose the output of my model is with shape (None,1) and how can I define custom loss function such as
```
def mycrossentropy(y_true, y_pred):
    loss = K.categorical_crossentropy(K.exp(y_true)/K.sum(K.exp(y_true)), K.exp(y_pred)/K.sum(K.exp(y_pred)))
    return loss
```
This function is used for normal tensor with shape (20,1) before. Now with new model, the output shape become (None, 1) ragged tensor. So, in this function y_pred should be a ragged tensor with shape(None, 1) and I can guarantee that y_true and y_pred have the same dimensions during training.

I am using tf 2.5.2 and tf.keras API for model define

**Comment by mohantym** [comment]
Are you satisfied with the resolution of your issue?
[Yes](https://goo.gl/forms/Oe0tEvODFRoI2gJF3)
[No](https://goo.gl/forms/fUjzOfrtkFbrOT8d2)
